recipes-samples/images/lmp-gateway-image.bb

```diff
@@ -12,17 +12,16 @@ require lmp-feature-sbin-path-helper.inc
 require lmp-feature-sysctl-hang-crash-helper.inc
 require lmp-feature-sysctl-net-queue-pfifo-fast.inc
 
+require ${@bb.utils.contains('MACHINE_FEATURES', 'optee', 'lmp-feature-optee.inc', '', d)}
+
 BT_6LOWPAN_NETWORK = "fe80:0:0:0:d4e7::1/80"
 require lmp-feature-bt-6lowpan.inc
 
 require lmp-service-bluetooth-disable.inc
-require lmp-service-ostree-pending-reboot.inc
+require ${@bb.utils.contains('SOTA_CLIENT', 'aktualizr', 'lmp-service-ostree-pending-reboot.inc', '', d)}
```

Comment:
Thank you for cleaning this up.  It was on my discussion list to talk about the fact that we just added this and now it's semi-obsolete for aktualizr-lite.